lib/node_modules/@stdlib/stats/base/ndarray/dmeanstdev/README.md

```diff
@@ -0,0 +1,153 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# dmeanstdev
+
+> Compute the [arithmetic mean][arithmetic-mean] and [standard deviation][standard-deviation] of a one-dimensional double-precision floating-point ndarray.
+
+<section class="intro">
+
+The [arithmetic mean][arithmetic-mean] and [standard deviation][standard-deviation] are defined as
```

Comment:
You should just copy the introduction from `stats/strided/dmeanstdev` instead of rolling your own. This makes find-and-replace easier when we are consistent in our docs across similar packages.